grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcService.java

```diff
@@ -62,6 +68,18 @@ static GrpcServiceBuilder builder() {
      */
     List<ServerServiceDefinition> services();
 
+    /**
+     * Returns a {@link Map} whose key is a route path and whose value is {@link MethodDescriptor}, which is
+     * serviced by this service.
+     */
+    default Map<String, MethodDescriptor<?, ?>> methods() {
```

Comment:
Does it make sense to return `Map<String, ServiceMethodDefinition<?, ?>>` instead? Then maybe we can fetch from `HandlerRegistry` without any stream processing?

Reply:
Oh, I don’t think of that. It is nice idea!